src/Views/EncryptView.vala

```diff
@@ -148,6 +155,21 @@ public class EncryptView : AbstractInstallerView {
             confirm_entry.is_valid = confirm_password ();
             update_next_button ();
         });
+
+        // Forward keys so we can type immediate when list is focused
+        var key_controller = new Gtk.EventControllerKey ();
+        key_controller.key_pressed.connect ((keyval, keycode, state) => {
+            if (keyval <= 65000) {
```

Comment:
What is this number? Shouldn't it be 65535? Maybe move it to constant

Reply:
Do you know of a good existing constant for this? I was trying to find the line between navigation keys and characters

Reply:
That's way better, thank you!